internal/tools/securitypolicy/main.go

```diff
@@ -83,21 +83,21 @@ type EnvironmentVariableRule struct {
 	Rule     string `toml:"rule"`
 }
 
-type Image struct {
+type Container struct {
 	Name     string                    `toml:"name"`
-	Auth     ImageAuth                 `toml:"auth"`
+	Auth     ContainerAuth             `toml:"auth"`
 	Command  []string                  `toml:"command"`
 	EnvRules []EnvironmentVariableRule `toml:"env_rule"`
 }
 
-type ImageAuth struct {
+type ContainerAuth struct {
```

Comment:
nit: would `AuthConfig` work better here?

Reply:
I did ContainerAuth to indicate what the Auth is for `AuthConfig` doesn't indicate what Auth. This is an auth that "belongs" to the Container.

I don't have strong opinions.

After hearing my explanation, what do you think is best?

Reply:
I thought about this a bit more. I think it should stay `ImageAuth` as its the auth info to download the image for the container.

Thoughts?

Reply:
`ImageAuth` works too.

Reply:
Done